src/cmd/cmd.zig

```diff
@@ -85,15 +82,23 @@ var gossip_port_option = cli.Option{
 var repair_port_option = cli.Option{
     .long_name = "repair-port",
     .help = "The port to run tvu repair listener - default: 8002",
-    .value_ref = cli.mkRef(&config.current.repair.port),
+    .value_ref = cli.mkRef(&config.current.tvu.repair_port),
     .required = false,
     .value_name = "Repair Port",
 };
 
+var tvu_port_option = cli.Option{
```

Comment:
This port is part of the external API of a solana validator so it might be good to use the standard name. This port is exposed publicly in gossip as SOCKET_TAG_TVU. But technically that's just a const name in the rust code. It's actually serialized as the number 10 in ContactInfo, so we could technically give it a new name. It feels like a grey area to me.

Reply:
i feel like the rename from tvu -> turbine is a 10x improvement in readability - as long as we have proper references and documentation that turbine_port == tvu_port then i think were ok (similar to how were renaming the structs with proper references + hardlinks)

the way i look at the name changes - if we dont change it were just taking on tech debt from the rust client (imagine how expensive it would be for them to change tvu -> turbine lol)

Reply:
I'm not sure how I feel about calling this the "turbine" port, but I can't think of a better name. So I just went ahead with your suggestion: a930874c7ab3cab8ac60730da9994f5ec847ab64

Anyway, these are my scattered thoughts...

I get that the lack of a component called "tvu" in sig means the name is not as ideal as it is in agave. But I also feel that "turbine" and "tvu" are both esoteric terms that require special knowledge of solana to make sense of them, so I don't see a huge difference in intelligibility between them.

One concern with turbine is that it is a general mechanism which in theory can be used for things other than shreds. I've heard some talk about using turbine as the mechanism to distribute gossip data, for example.

The other issue with turbine is that it describes the entire mechanism of both sending and receiving, whereas this is only the port where we receive items over turbine. Most of the actual logic that's specific to the turbine mechanism is actually in the retransmit stage. Retransmit uses a different port number than this. Though that is assigned by the OS, so there's not really a meaningful port number to keep track of over there.

Anyway I don't see these as huge issues, but just playing devil's advocate. For any name I can think of, I can imagine potential sources of confusion.

Reply:
yeah those are great points - the way i see it is, though turbine is not perfect, its still better than tvu

and though theres been talk of using turbine elsewhere, i think thats very far away on the roadmap and we dont need to worry about it now

for the second point, maybe `turbine_recv_port` would be better ?

Reply:
99ff6cda5103d2a279977630f06e7cba2c0932ea